packages/botonic-nlp/tests/utils/model/model.json

```diff
@@ -0,0 +1 @@
ro":null,"input_length":12,"name":"EmbeddingsLayer","trainable":true,"batch_input_shape":[null,12]},"inbound_nodes":[[["InputLayer",0,0,{}]]]},{"name":"DropoutLayer","class_name":"Dropout","config":{"rate":0.1,"noise_shape":null,"seed":null,"name":"DropoutLayer","trainable":true},"inbound_nodes":[[["EmbeddingsLayer",0,0,{}]]]},{"name":"BidirectionalLayer","class_name":"Bidirectional","config":{"merge_mode":"concat","layer":{"class_name":"LSTM","config":{"name":"lstm_LSTM1","trainable":true,"units":128,"activation":"tanh","recurrent_activation":"hard_sigmoid","use_bias":true,"kernel_initializer":{"class_name":"VarianceScaling","config":{"scale":1,"mode":"fan_avg","distribution":"normal","seed":null}},"recurrent_initializer":{"class_name":"Orthogonal","config":{"gain":1,"seed":null}},"bias_initializer":{"class_name":"Zeros","config":{}},"unit_forget_bias":null,"kernel_regularizer":null,"recurrent_regularizer":null,"bias_regularizer":null,"activity_regularizer":null,"kernel_constraint":null,"recurrent_constraint":null,"bias_constraint":null,"dropout":0,"recurrent_dropout":0.10000000149011612,"implementation":null,"return_sequences":true,"return_state":false,"go_backwards":false,"stateful":false,"unroll":false}},"name":"BidirectionalLayer","trainable":true},"inbound_nodes":[[["DropoutLayer",0,0,{}]]]},{"name":"TimeDistributedLayer","class_name":"TimeDistributed","config":{"layer":{"class_name":"Dense","config":{"units":5,"activation":"softmax","use_bias":true,"kernel_initializer":{"class_name":"VarianceScaling","config":{"scale":1,"mode":"fan_avg","distribution":"normal","seed":null}},"bias_initializer":{"class_name":"Zeros","config":{}},"kernel_regularizer":null,"bias_regularizer":null,"activity_regularizer":null,"kernel_constraint":null,"bias_constraint":null,"name":"dense_Dense1","trainable":true}},"name":"TimeDistributedLayer","trainable":true},"inbound_nodes":[[["BidirectionalLayer",0,0,{}]]]}],"input_layers":[["InputLayer",0,0]],"output_layers":[["TimeDistributedLayer",0,0]]},"keras_version":"tfjs-layers 2.8.5","backend":"tensor_flow.js"},"weightsManifest":[{"paths":["weights.bin"],"weights":[{"name":"EmbeddingsLayer/embeddings","shape":[57,50],"dtype":"float32"},{"name":"BidirectionalLayer/forward_lstm_LSTM1/kernel","shape":[50,512],"dtype":"float32"},{"name":"BidirectionalLayer/forward_lstm_LSTM1/recurrent_kernel","shape":[128,512],"dtype":"float32"},{"name":"BidirectionalLayer/forward_lstm_LSTM1/bias","shape":[512],"dtype":"float32"},{"name":"BidirectionalLayer/backward_lstm_LSTM1/kernel","shape":[50,512],"dtype":"float32"},{"name":"BidirectionalLayer/backward_lstm_LSTM1/recurrent_kernel","shape":[128,512],"dtype":"float32"},{"name":"BidirectionalLayer/backward_lstm_LSTM1/bias","shape":[512],"dtype":"float32"},{"name":"TimeDistributedLayer/kernel","shape":[256,5],"dtype":"float32"},{"name":"TimeDistributedLayer/bias","shape":[5],"dtype":"float32"}]}],"format":"layers-model","generatedBy":"TensorFlow.js tfjs-layers v2.8.5","convertedBy":null}
```

Comment:
does the model apply to both booking and shopping? create a README at that folder if you need to explain this or something else